src/platform/linux/wayland/event_loop.rs

```diff
@@ -476,20 +476,136 @@ fn seat_implementation() -> wl_seat::Implementation<SeatIData> {
                     kbd.release();
                 }
             }
-            // create touch if applicable
-            if capabilities.contains(wl_seat::Capability::Touch) && idata.touch.is_none() {
-                let touch = seat.get_touch().expect("Seat is not dead");
-                let t_idata = super::touch::TouchIData::new(
-                    &idata.sink,
-                    idata.windows_token.clone()
+            // TODO: Handle touch
+        }
+    }
+}
+
+struct PointerIData {
+    sink: Arc<Mutex<EventsLoopSink>>,
+    windows_token: StateToken<WindowStore>,
+    mouse_focus: Option<WindowId>,
+    axis_buffer: Option<(f32, f32)>,
+    axis_discrete_buffer: Option<(i32, i32)>,
+    axis_state: TouchPhase,
+}
```

Comment:
Is it github messing up the displayed diff? It looks like you're undoing part of #342 by re-adding pointer logic in `event_loop.rs` and removing touch support in the seat...

Reply:
Woops I think I grabbed the wrong pieces while resolving this, thanks and good catch.  Let me try again.